emailvalidation.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+require 'apiInfo.php';
+
+/**
+ * User's email address is validating here by calling an api and return true or false.
+ *
+ * @param string $emailAddress
+ *   email address taking as input.
+ */
+function emailVal($emailAddress): bool {
+	global $api_key;
+	$apikey = $api_key;
+	$curl = curl_init();
+	$url = "https://emailvalidation.abstractapi.com/v1/?api_key=". $apikey. "&email=" . $emailAddress;
+
+	$ch = curl_init();
```

Comment:
[This feedback](https://github.com/nazar-innoraft/InternshipPHPTask/pull/1#discussion_r1507153998) still not addressed.

Reply:
Done.